osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -168,11 +168,13 @@ protected override void LoadComplete()
                 if (healthBar.NewValue)
                 {
                     HealthDisplay.FadeIn(fade_duration, fade_easing);
+                    FailingLayer.FadeIn(fade_duration, fade_easing);
```

Comment:
This should be inside of `FailingLayer` itself to have all the visibility/setting logic in one place.